mcmd/commands/enable.py

```diff
@@ -35,3 +46,26 @@ def enable_rls(args):
     ensure_resource_exists(args.entity, ResourceType.ENTITY_TYPE)
     post(config.api('rls'), data={'id': args.entity,
                                   'rlsEnabled': True})
+
+
+@login
+def enable_theme(args):
+    """
+    enable_theme enables a bootstrap theme
+    :param args: commandline arguments containing the id of the theme (without .css)
+    :exception McmdError: when applying the theme fails
+    :return None
+    """
+    theme = args.theme.replace('.css', '').replace('.min', '')
+    io.start('Applying theme {}'.format(highlight(theme)))
+    # Resource can be bootstrap-name.min.css (if molgenis theme), or name.min.css (if uploaded .min.css), or
+    # name.css (if uploaded as .css).
+    if one_resource_exists([theme + '.min.css', theme + '.css', 'bootstrap-' + theme + '.min.css'], ResourceType.THEME):
+        # Molgenis themes start with bootstrap- but this is stripped from the name in the theme manager
+        try:
+            post(config.api('set_theme'), theme)
+        except:
+            post(config.api('set_theme'), theme.split('bootstrap-')[1])
+    else:
+        raise McmdError(
+            'Applying theme failed. Does theme [{}] exist in the [{}] table?'.format(args.theme, 'sys_set_StyleSheet'))
```

Comment:
```suggestion
            'Applying theme failed. No themes found containing {} in the name'.format(args.theme))
```

Because if I enter:
```
mcmd enable theme mytheme
```

It could actually mean 'bootstrap-mytheme.css', 'mytheme.min.css', etc.